contract/src/orca.contract.js

```diff
@@ -7,8 +7,14 @@ import { Fail } from '@endo/errors';
 import { E } from '@endo/far';
 import { M } from '@endo/patterns';
 import * as flows from './orca.flows.js';
-
-import '@agoric/zoe/src/zoeService/types-ambient.js';
+// import {ZCF} from "@agoric/zoe/src/contractFacet/types-ambient.js"
+// import '@agoric/zoe/src/zoeService/types-ambient.js';
+// import '@agoric/zoe/src/contractFacet/types-ambient.js';
+// import '@agoric/vats/src/core/types-ambient.js';
+// import '@agoric/vow/src/types.js';
+// import { Remote } from '@agoric/internal/src/types.js';
+// import '@agoric/orchestration/src/types.js';
+// import '@agoric/orchestration/src/exos/int'
```

Comment:
Please strike all this commented out code or add a comment about why it should be there